Language/Generics/Superbounded_types/well_bounded_A01_t04.dart

```diff
@@ -16,10 +16,17 @@ class B1<X extends A<Object?>> {}
 class B2<X extends A<dynamic>> {}
 class B3<X extends A<void>> {}
 class B4<X extends A<Never>> {}
+class B5<X extends A<X>> {}
+
+mixin M<X extends A<X>> on A<X> {}
```

Comment:
Same here, but in order to cover mixins we could do this:
```suggestion
mixin M<X extends A<Object?>> on A<Never> {}
```